src/components/Graphs/index.jsx

```diff
@@ -61,7 +61,7 @@ const Graphs = ({
         <div key={title}>
           <h2 className={classes.benchmarkTitle}>{title}</h2>
           <a href={jointUrl} target="_blank" rel="noopener noreferrer"><LinkIcon className={classes.linkIcon} /></a>
-          <a href={`/${platform}/${benchmark}`} rel="noopener noreferrer"><ArrowDownward className={classes.linkIcon} /></a>
+          {overviewMode ? <a href={`/${platform}/${benchmark}`} rel="noopener noreferrer"><ArrowDownward className={classes.linkIcon} /></a> : null }
```

Comment:
Hi @armenzg, 
As you asked, I have passed `benchmark ` and this benchmark variable is returning 'overview' value for all benchmarks on the overview page. 
In my understanding here should come the value/key of benchmark which should be different for all benchmarks. And after clicking the link it should take to `platform/benchmark`. 
Please let me know if I am missing something here.

Reply:
Let me pull your code and see what other option we have.